src/data/data1d/filter1d/utils/padDataToNextPowerOfTwo.ts

```diff
@@ -0,0 +1,48 @@
+import { xSequentialFill } from 'ml-spectra-processing';
+
+import { Datum1D } from '../../../types/data1d';
+
+export function padDataToNextPowerOfTwo(
+  datum1D: Datum1D,
+  digitalFilterApplied: boolean,
```

Comment:
There is something I don't understand here. When is the digital filter applied and how does it impact this code. Because in the list of filters it appears at the beginning and I would not expect that it impacts further steps.

![image](https://user-images.githubusercontent.com/1484241/187016557-85fcd40d-4143-41e3-ba6e-d80f43c24398.png)